pkg/controllers/deprovisioning/events/events.go

```diff
@@ -60,14 +60,14 @@ func Terminating(node *v1.Node, machine *v1alpha5.Machine, reason string) []even
 			InvolvedObject: node,
 			Type:           v1.EventTypeNormal,
 			Reason:         "DeprovisioningTerminating",
-			Message:        fmt.Sprintf("Deprovisioning node via %s", reason),
+			Message:        fmt.Sprintf("Deprovisioning node via: %s", reason),
```

Comment:
Now that you have `:`, does it make sense to drop `node via`

`Deprovisioning: Replace`
`Deprovisioning: Delete`

Reply:
Eh, I think contextualizing the message still makes sense with `Deprovisioning`. You see this with `Node` too:

```console
default     24s         Normal    NodeNotSchedulable        node/ip-192-168-96-64.us-west-2.compute.internal     Node ip-192-168-96-64.us-west-2.compute.internal status is now: NodeNotSchedulable
default     23s         Warning   Terminating               node/ip-192-168-96-64.us-west-2.compute.internal     Node ip-192-168-96-64.us-west-2.compute.internal event: Node is terminating
default     20s         Normal    RemovingNode              node/ip-192-168-96-64.us-west-2.compute.internal     Node ip-192-168-96-64.us-west-2.compute.internal event: Removing Node ip-192-168-96-64.us-west-2.compute.internal from Controller
```

Reply:
Not blocking for now, but I'm not sure we need to follow the pattern -- there may be other reasons it's overly verbose (compatibility)